### PR TITLE
webpack and scss

### PR DESCRIPTION
**Pull request by JakubSedlacik**
Code review needed. Thanks. @Machy8

### CODE REVIEW

File: assets/scss-src/_footer.scss [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+.instaContainer {
+    display: flex;
+    flex-flow: nowrap;
+    justify-content: center;
+    margin: auto;
+    max-width: 1000px;
+    padding: 0 30px 0 30px;
+    overflow: hidden;
+
+    @include breakpoint( 1040px) {
```
Comment (Machy8) [inline assets/scss-src/_footer.scss:45]: Unnecessary space before the size. Fix it in the whole file

File: assets/scss-src/_footer.scss [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+    
+        @include breakpoint( 600px) {
+            max-width: 200px;
+            width: 100%;
+        }
+    }
+} 
+  
+.instaIcon {
+    opacity: 0%;
```
Comment (Machy8) [inline assets/scss-src/_footer.scss:102]: Unnecessary percentages

File: assets/scss-src/_footer.scss [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+    font-variant: normal;
+    text-transform: none;
+    line-height: 1;
+  
+    /* Better Font Rendering =========== */
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+}
+  
+  .icon-search:before {
```
Comment (Machy8) [inline assets/scss-src/_footer.scss:146]: Wrong indentation

File: assets/scss-src/_intro.scss [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+  top: -38px;
+  left: 180px;
+
+  @include breakpoint( 900px) {
+    position: relative;
+    top: -30px;
+    left: 145px;
+  }
+}
+
```
Comment (Machy8) [inline assets/scss-src/_intro.scss:65]: Missing empty line on the end of the file

File: assets/scss-src/_list-of-articles.scss [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+  margin-top: 70px;
+  padding: 14px 0 10px 0;
+  width: 200px;
+
+  &:hover {
+    background-color: $dark-blue;
+    color: rgb(231, 231, 231);
+  }
+}
+
```
Comment (Machy8) [inline assets/scss-src/_list-of-articles.scss:115]: Missing empty line on the end of the file

File: assets/scss-src/_maintiles.scss [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+        @include breakpoint( 900px) {
+           max-width: 320px;
+            width: 100%;   
+        }
+    }
+
+    a {
+        display: block;
+        text-decoration: none;
+        color: rgb(255, 255, 255);
```
Comment (Machy8) [inline assets/scss-src/_maintiles.scss:43]: This could be #fff or white

File: assets/scss-src/_nav.scss [hunk 7]
```diff
@@ -0,0 +1,238 @@
...
+
+  .iconRow{
+    a {
+      text-decoration: none;
+      color: $lightish-grey;
+      margin: 0;
+      padding: 10px 5px 10px 5px;
+
+      &:hover {
+        color: rgb(73, 73, 73);
```
Comment (Machy8) [inline assets/scss-src/_nav.scss:21]: Do not use rgb when not necessary. In the browser, in the css panel, there is a switch in computed properties to see the hex #xxx color.

File: assets/scss-src/_nav.scss [hunk 8]
```diff
@@ -0,0 +1,238 @@
...
+/* FIRST NAV BAR */
+#navContainer {
+  display: flex;
+  height: 56px;
+  justify-content: space-around;
+  max-width: 1180px;
+  margin: auto;
+  align-items: center;
+  text-align: center;
+  background-color: rgba(255, 255, 255, 0.952);
```
Comment (Machy8) [inline assets/scss-src/_nav.scss:36]: Use full numbers. 0.95 or 96.

File: assets/scss-src/_nav.scss [hunk 9]
```diff
@@ -0,0 +1,238 @@
...
+
+  #navLogo {
+    position: relative;
+    left: -10px;
+    padding: 8px 7px 2px 7px;
+  }
+}
+
+#navMain {
+  position: relative; /* FOR ABSOLUTE POSITIONING OF DROPDOWN ITEMS */
```
Comment (Machy8) [inline assets/scss-src/_nav.scss:50]: The comment can be removed

File: assets/scss-src/_nav.scss [hunk 10]
```diff
@@ -0,0 +1,238 @@
...
+
+  a {
+    background-color: rgb(224, 224, 224);
+  }
+}
+
+/* DROPDOWN HOVER */
+#ddLink-1:hover {
+  visibility: visible;
+}
```
Comment (Machy8) [inline assets/scss-src/_nav.scss:109]: Keep lines indented in the same way. In this section, one is one not. Add an empty line behind this bracket or remove them behind the next sections.

File: assets/scss-src/_nav.scss [hunk 11]
```diff
@@ -0,0 +1,238 @@
...
+    }
+
+    i {
+      font-size: 1.6em;
+      position: relative;
+      top: 2px;
+    }
+  }
+}
+
```
Comment (Machy8) [inline assets/scss-src/_nav.scss:232]: Keep only one empty line on the end of the file.

File: assets/scss-src/_universal.scss [hunk 12]
```diff
@@ -0,0 +1,22 @@
+* {
```
Comment (Machy8) [inline assets/scss-src/_universal.scss:1]: Rename this file to something like `common|layout`. It is more commonly used then universal.

File: assets/scss-src/helpers/_vars.scss [hunk 13]
```diff
@@ -0,0 +1,5 @@
+$breakpoint-default-feature: max-width;
+$dark-blue: rgb(4, 0, 22);
+$light-grey: rgba(0, 0, 0, 0.1);
+$lightish-grey: rgba(0, 0, 0, 0.4);
+$darker-grey: rgba(0, 0, 0, 0.65);
```
Comment (Machy8) [inline assets/scss-src/helpers/_vars.scss:5]: Missing empty line on the end of the file

File: assets/scss-src/index.scss [hunk 14]
```diff
@@ -0,0 +1,9 @@
+@import '../../node_modules/breakpoint-sass/stylesheets/breakpoint';
```
Comment (Machy8) [inline assets/scss-src/index.scss:1]: The `../../node_modules/` can be replaced by `~` https://github.com/PatrickJS/starter/issues/727

File: index.html [hunk 15]
```diff
@@ -33,11 +32,11 @@
                <li class="navMainLink"><a href="#9">slunce</a></li>
             </ul>
 
-            <div class="firstNavIcons iconRow">   
-                <a href="#" class="icon-facebook2"></a>
-                <a href="#" class="icon-youtube"></a>   
-                <a href="#" class="icon-twitter"></a>   
-                <a href="#" class="icon-firefox"></a>   
+            <div class="iconRow">   
+                <a href="#" i class="icon-facebook2"></a>
```
Comment (Machy8) [inline index.html:36]: What is the `i` doing here?

File: assets/scss-src/_intro.scss (hunk 4 above)
Comment (JakubSedlacik) [reply assets/scss-src/_intro.scss:65]: Line #65 is empty :).

File: webpack.config.js [hunk 16]
```diff
@@ -0,0 +1,26 @@
...
+        rules: [
+            {
+            test: /\.(scss)$/,
+            use: [MiniCssExtractPlugin.loader, 'css-loader', 'sass-loader']
+            },
+            {
+            test: /\.(png|jpg|eot|svg|woff|ttf)$/,
+            use: 'url-loader', // může taky použít file loader...ten funguje taky
+            }
+        ]
```
Comment (Machy8) [inline webpack.config.js:24]: Remove the block with url and replace css with this
```
{
	test: /\.(scss)$/,
	use: [
		MiniCssExtractPlugin.loader,
		{
			loader: 'css-loader',
			options: { url: false }
		},
		'sass-loader'
	]
}
```

**Pull request merged**